third_party/xla_client/BUILD

```diff
@@ -360,7 +359,6 @@ cc_library(
         "@org_tensorflow//tensorflow/compiler/xla:types",
         "@org_tensorflow//tensorflow/core:lib",
```

Comment:
same question, do we need this `tensorflow/core` deps directly or generally?

we would migrate to pull XLA from OpenXLA, and would delete tensorflow too, so would delete `tensorflow/core` deps too

Reply:
same as above.